qiskit/pulse/schedule.py

```diff
@@ -397,13 +398,12 @@ def __lshift__(self, time: int) -> 'Schedule':
         return self.shift(time)
 
     def __repr__(self):
-        res = 'Schedule("name=%s", ' % self._name if self._name else 'Schedule('
-        res += '%d, ' % self.start_time
-        instructions = [repr(instr) for instr in self.instructions]
-        res += ', '.join([str(i) for i in instructions[:50]])
-        if len(instructions) > 50:
-            return res + ', ...)'
-        return res + ')'
+        return "Schedule(name={}, t=[{}, {}], n_instructions={})".format(
```

Comment:
For other use cases besides the error message you are improving, you'd actually want to see the instructions. I'd be very, very happy if you fixed up this method to use `format` rather than `%`, but let's keep the output the same.